include/dd/Package.hpp

```diff
@@ -2178,22 +2178,22 @@ namespace dd {
             return r;
         }
 
-        std::map<std::string, double> getProbVectorFromDensityMatrix(dEdge e, double measurementThreshold) {
-            std::map<std::string, double> measuredResult = {};
-            double                        p0, p1;
-            double                        globalProbability;
-            double                        statesToMeasure;
+        std::map<std::string, dd::fp> getProbVectorFromDensityMatrix(dEdge e, double measurementThreshold) {
+            std::map<std::string, dd::fp> measuredResult = {};
+            dd::fp                        p0, p1, globalProbability;
+            dEdge::alignDensityEdge(e);
+            if (static_cast<unsigned long long>(2ULL << (e.p->v + 1)) >= std::numeric_limits<unsigned long long>::max()) {
```

Comment:
This check can, per definition, never return `true`.
In this case, `std::pow(2, e.p->v+1)` and `static_cast<double>(std::numeric_limits<unsigned long long>::max())` have to be used.

Reply:
Indeed! I changed that as suggested.